src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -380,6 +380,19 @@ class ContentExplorer extends Component {
         return <ContentExplorerEmptyState isSearch={isViewingSearchResults} />;
     };
 
+    getUpdatedChooseButtonProps = chooseButtonProps => {
```

Comment:
I think this method is better suited in `src/features/content-explorer/content-explorer/ContentExplorerActionButtons.js`.